### ISSUE TITLE
Account Falsely Blocked Due To “Abusive Activity”

### ISSUE THREAD

**Issue by AshwinFernandez** [googlecolab/colabtools#4985]
Hi,

I am reaching out because my account has been blocked, and I am unable to access Google Colab. As a paying customer and a student with a project due in just one day, this is incredibly frustrating.

Could you please explain why my account is blocked and help me regain access as soon as possible? I rely on your service for my work, and I need to resolve this issue urgently.

Thank you for your prompt attention to this matter.

**Comment by GiGiGitHubye** [comment]
Same issue when training Yolo11, get banned

**Comment by buriburix5** [comment]
I got banned too, what have you done google

**Comment by HARSHEREX-ACX** [comment]
need immidiate resolution just subscribed to pro+ 3 hours ago and got this issue. what a great service

**Comment by marcory-hub** [comment]
Same here! Same issue when tuning training Yolo11 with wandb

**Comment by inanbilal** [comment]
yolo11 eğitiminde banlandım.

**Comment by buriburix5** [comment]
I think I found the reason
https://github.com/ltdrdata/ComfyUI-Impact-Pack/issues/843

**Comment by FurkanGozukara** [comment]
related to Ultralytics  package exploit this is big

**Comment by jandolina** [comment]
Ultralytics 8.3.41 and 8.3.42 are compromised
If you use an venv environment enable it
run `pip show ultralytics`
If is shows either version above roll back to the last 'safe' version

`pip install --force-reinstall -v "ultralytics==8.3.40"`

**Comment by cperry-goog** [comment]
Yesterday ultralytics released 8.3.41, which contained malicious code that could be used for cryptocurrency mining. Automated Colab abuse detection picked this up and began banning unsuspecting ultralytics users who installed 8.3.41 and started training with that version.

We noticed this uptick in bans, but we had made no recent changes to our configs, so were unable to immediately diagnose the issue. Late in the day our Backend TL hopped on while on vacation and root-caused the issue and reported it upstream: https://github.com/ultralytics/ultralytics/issues/18027 - ultralytics subsequently removed 8.3.41 (and briefly, 8.3.42) from PyPI.

We have unbanned all users impacted yesterday, and this issue should be resolved. A big thank you to all users who reported this. We'll have an internal post-mortem to see if we can better avoid these types of attacks in the future.
